client/src/components/tabs.js

```diff
@@ -85,16 +85,16 @@ export default class Tabs extends React.Component{
                     This is based on the historical growth data of each country that is currently ahead of {country_name} in the outbreak.
                     </p>
                     <p className="is-size-6">
-                    Viewing this can offer unique insights into the range of possible outcomes. Not based on epidemiological models, only on historical data experienced by other countries.
+                    Viewing this can offer unique insights into the range of possible outcomes. The forecast is not based on epidemiological models, just on historical data experienced by other countries.
```

Comment:
I think the previous form wasn't a completed sentence, even though the meaning was clear.